internal/main.go

```diff
@@ -48,6 +48,22 @@ const (
 	cmdlineOEMFlag = "coreos.oem.id"
 )
 
+type StringMapFlag map[string]string
+
+func (sm *StringMapFlag) String() string {
+	return fmt.Sprintf("%+v", *sm)
+}
+
+func (sm *StringMapFlag) Set(v string) error {
+	vals := strings.Split(v, "=")
+	if len(vals) != 2 {
+		fmt.Println("Could not split override set correctly")
```

Comment:
This is potentially a user-visible error message and should clearly describe the problem.